### ISSUE TITLE
Provide filter on Default VPC in VPC Screen

### ISSUE THREAD

**Issue by tomvachon** [RiotGames/cloud-inquisitor#167]
We don't really use the default VPC, it would be great to have a filter which could remove them on the VPC screen

**Comment by TheM0ng00se** [comment]
Yep, same here...read my mind as I was planning on this a few weeks ago :)
